Makefile

```diff
@@ -335,26 +335,58 @@ shadow-server: export TARGET := clojure
 shadow-server:##@ Start shadow-cljs in server mode for watching
 	yarn shadow-cljs server
 
-test-watch: export TARGET := clojure
-test-watch: ##@ Watch tests and re-run no changes to cljs files
-	yarn install
-	nodemon --exec 'yarn shadow-cljs compile mocks && yarn shadow-cljs compile test && node --require ./test-resources/override.js target/test/test.js' -e cljs
+_clojure-test: export TARGET := clojure
+_clojure-test:
+	yarn install && \
```

Comment:
Why does this start with underscore? And it's missing a help message.

Reply:
And the name should be `test-clojure` if we want to have consistent prefixes.

Reply:
> Why does this start with underscore? And it's missing a help message.

There are other targets prefixed with underscore, so I used the same idea to mean "private" targets that shouldn't be used directly.

> And the name should be test-clojure if we want to have consistent prefixes.

Good point, will do.